smart-contract-verifier/smart-contract-verifier-http/src/verification_response.rs

```diff
@@ -3,14 +3,36 @@ use serde::{Deserialize, Serialize};
 use smart_contract_verifier::{SourcifySuccess, VerificationSuccess};
 use std::{collections::BTreeMap, fmt::Display};
 
-#[derive(Debug, Deserialize, Serialize, PartialEq, Eq)]
+#[derive(Clone, Debug, Deserialize, Serialize, PartialEq, Eq)]
 pub struct VerificationResponse {
     pub message: String,
     pub result: Option<VerificationResult>,
     pub status: VerificationStatus,
 }
 
-#[derive(Debug, Deserialize, Serialize, PartialEq, Eq)]
+#[derive(Clone, Debug, Deserialize, Serialize, PartialEq, Eq)]
+#[serde(tag = "type")]
+pub enum BytecodePart {
+    Main { data: DisplayBytes },
```

Comment:
You can skip this idea but I suggest add `order` field for Main and Meta, that will represent position in local bytecode

I understand that it can be obtained from the position of part in array, but I think it will be more visual and informative and doesn't require a lot of efforts from our side

Reply:
I personally prefer not to have `order` as it would make it more difficult to parse it both on the client and the server side. Besides that, it potentially opens a possibility for some indexes to be missed or to be equal (e.g, 1, 2, 2, 4), while position in an array is always consecutive